playground/ty/vite.config.ts

```diff
@@ -15,15 +15,15 @@ const PYODIDE_EXCLUDE = [
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react(), tailwindcss(), viteStaticCopyPyodide()],
-  optimizeDeps: { exclude: ["pyodide"] },
+  optimizeDeps: { exclude: ["pyodide", "ty_wasm"] },
 });
 
 export function viteStaticCopyPyodide() {
   const pyodideDir = dirname(fileURLToPath(import.meta.resolve("pyodide")));
   return viteStaticCopy({
     targets: [
       {
-        src: [join(pyodideDir, "*"), ...PYODIDE_EXCLUDE],
+        src: [join(pyodideDir, "*").replace(/\\/g, "/"), ...PYODIDE_EXCLUDE],
```

Comment:
This makes little sense to me, given that `path.join` uses the platform-specific path separator. Which path contains the `/` where windows expects a `\`?

Reply:
Right, this is a glob and not a path. 

I suggest then to call `normalizePath(pyodideDir)` (or normalize before assigning to `pyodideDir`) to scope the normalize exactly to where it's necessary.